app/helpers/releases_helper.rb

```diff
@@ -8,6 +8,17 @@ def release_label(project, release)
     )
   end
 
+  def status_glyphicon(status_state)
+    icon, text = case status_state
+           when "success" then ["ok", "success"]
+           when "failure" then ["remove", "danger"]
+           when "missing" then ["minus", "muted"]
+           when "pending" then ["hourglass", "primary"]
+           end
+
+    %(<span class="glyphicon glyphicon-#{icon} text-#{text}" aria-hidden="true"></span>).html_safe
```

Comment:
The problem is that `icon_tag` doesn't seem to support setting additional classes, since it overwrites the `class` attribute. I frankly don't think it's necessary, since this is already declarative Bootstrap code.